### DISCUSSION TITLE
A little problem in FEPX Reference Manual

### DISCUSSION THREAD

**Discussion by kwche** [neperfepx/FEPX#67] [Q&A] [answered]
In Page 9 Line 1 (_2.1.1.1 Model Description_), the unit of **_gammadot_0_**, the fixed-rate strain rate scaling coefficient, is said to be **[force/area]**, which is **MPa**. But actually it should be **s<sup>-1</sup>**,  right?
![image](https://user-images.githubusercontent.com/65537204/148727899-95cd83c0-5738-462c-b2e6-45caa45a5d34.png)

**Comment by mkasemer** [comment, marked as answer]
@kwche - You are correct about the units. We are currently overhauling the documentation, and the fix will be made in the next public release. Thank you for reporting this error!
